packages/fluentui/react-northstar/src/components/Carousel/Carousel.tsx

```diff
@@ -157,6 +186,10 @@ export const Carousel = (React.forwardRef<HTMLDivElement, CarouselProps>((props,
     styles,
     variables,
     disableClickableNav,
+    animationEnterFromLeft,
+    animationEnterFromRight,
+    animationExitToLeft,
+    animationExitToRight,
```

Comment:
Can we avoid `left` & `right` in naming? Use something like `start` & `end` for example.